File: 4153_Pythagoras/4153.cpp
// 피타고라스의 정리
#include <iostream>
#include <cstdio>
#include <cstdlib>
#include <cstring>
using namespace std;
int main(){
    int a, b, c;
    int num = 0;
    char buf[1024] = "";
    while(scanf("%d %d %d", &a, &b, &c) == 3){
        if (a == 0 && b == 0 && c == 0){
            break;
        }
        num += 1;
        
        int left = 0;
        int right = 0;

        if (a >= b && a >= c){
            left = a*a;
            right = b*b + c*c;
        }else if (b >= c && b >= a){
            left = b*b;
            right = a*a + c*c;
        }else if (c >= a && c >= b){
            left = c*c;
            right = a*a + b*b;
        }


        if (left == right){
            strcat(buf, "right\n");
        }else{
            strcat(buf, "wrong\n");
        }

    }
    printf("%s", buf);

}

File: 4153_Pythagoras/4153.py
# 피타고라스의 정리
import math
from sys import stdin
line = []
result = ""
while True:
  line = list(map(int, stdin.readline().split(" ")))
  
  if line[0] == 0 and line[1] == 0 and line[2] == 0:
    break
  
  if line[0] >= line[1] and line[0] >= line[2]:
    left = line[0] ** 2
    right = line[1]**2 + line[2]**2
  elif line[1] >= line[0] and line[1] >= line[2]:
    left = line[1] ** 2
    right = line[0]**2 + line[2]**2
  elif line[2] >= line[0] and line[2] >= line[1]:
    left = line[2] ** 2
    right = line[0]**2 + line[1]**2

  if left == right:
    result += "right\n"
  else:
    result += "wrong\n"

print(result)